src/Relation/Unary/Polymorphic/Properties.agda

```diff
@@ -0,0 +1,73 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of polymorphic versions of standard definitions in
+-- Relation.Unary
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Unary.Polymorphic.Properties where
+
+open import Level using (Level)
+open import Relation.Binary.Definitions hiding (Decidable; Universal)
+open import Relation.Nullary using (yes; no)
+open import Relation.Unary hiding (∅; U)
+open import Relation.Unary.Polymorphic
+open import Data.Unit.Polymorphic.Base using (tt)
+
+private
+  variable
+    a ℓ ℓ₁ ℓ₂ : Level
+    A : Set a
+
+----------------------------------------------------------------------
+-- The empty set
+
+∅? : Decidable {A = A} {ℓ} ∅
+∅? _ = no λ()
+
+∅-Empty : Empty {A = A} {ℓ} ∅
+∅-Empty _ ()
+
+∁∅-Universal : Universal {A = A} {ℓ} (∁ ∅)
+∁∅-Universal _ ()
+
+----------------------------------------------------------------------
+-- The universe
+
+U? : Decidable {A = A} {ℓ} U
+U? _ = yes tt
+
+U-Universal : Universal {A = A} {ℓ} U
+U-Universal _ = _
+
+∁U-Empty : Empty {A = A} {ℓ} (∁ U)
+∁U-Empty _ x∈∁U = x∈∁U _
+
+----------------------------------------------------------------------
+-- Subset properties
+
+∅-⊆ : (P : Pred A ℓ₁) → ∅ {ℓ = ℓ₂} ⊆ P
+∅-⊆ _ ()
+
+⊆-U : (P : Pred A ℓ₁) → P ⊆ U {ℓ = ℓ₂}
+⊆-U _ _ = _
+
+⊆-min : Minimum (_⊆_ {A = A} {ℓ}) ∅
```

Comment:
Might be cleaner to write this and the properties below as: `Minimum {A = Pred A ℓ} _⊆_ ∅`?

Reply:
:+1:

I also noticed that these can be generalised, e.g. `⊆-min : Min {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊆_ ∅`.